klaus/ctagsutils.py

```diff
@@ -41,5 +50,9 @@ def create_tagsfile(git_repo_path, git_rev):
 
 
 def delete_tagsfile(tagsfile_path):
-    """Delete a tagsfile."""
+    """Delete a tagsfile.
```

Comment:
This entire docstring is useless, maybe modulo the type (which should be a type hint)